topics/assembly/tutorials/assembly-with-preprocessing/tutorial.md

```diff
@@ -0,0 +1,629 @@
+> The downsampling parameters below have been chosen to have minimal impact on
+> the assembly results. Further speed-ups are certainly possible, but will
+> likely lead to different assembly outcomes.
+{: .comment}
+
+> ### {% icon hands_on %} Hands-on: Subsampling of paired-end short-reads data
+>
+> 1. **seqtk_sample** {% icon tool %} with the following parameters
+>    - {% icon param-files %} *"Input FASTA/Q file"*: The two datasets with the
+>      combined Illumina-sequenced forward and reverse reads, outputs of the
+>      first and the second run of **Collapse Collection**
+>    - *"RNG seed"*: 4
+>    - *"Subsample (decimal fraction or number)"*: 0.2
+>
+{: .hands_on}
+
+## Create assembly
+
+> ### {% icon hands_on %} Hands-on: Assembly of SARS-CoV2 genome
+>
+> 1. **Create assemblies with Unicycler** {% icon tool %} with the following parameters
+>    - *"Paired or Single end data?"*: `Paired`
+>      - {% icon param-file %} *"Select first set of reads"*: the combined
+>        Illumina-sequenced forward reads from all samples, output of the
+>        first **Collapse Collection** {% icon tool %} run (or first output of
+>        **seqtk_sample** {% icon tool %} for a subsample-based assembly)
+>      - {% icon param-file %} *"Select second set of reads"*: the combined
+>        Illumina-sequenced reverse reads from all samples, output of the
+>        second **Collapse Collection** {% icon tool %} run  (or second output
+>        of **seqtk_sample** {% icon tool %} for a subsample-based assembly)
+>
+>    - {% icon param-file %} *"Select long reads. If there are no long reads, leave this empty"*:
+>      Nanpore-sequenced reads from all samples, output of the third
+>      **Collapse Collection** {% icon tool %} run
+>    - *"Select Bridging mode"*: `Normal (moderate contig size and misassembly rate)`
+>    - *"Exclude contigs from the FASTA file which are shorter than this length (bp)"*:
+>      `10000`
+>    - *"The expected number of linear (i.e. non-circular) sequences in the assembly"*:
+>      `1`
+>
+{: .hands_on}
+
+## Explore assembly
+
+### Assembly inspection with Bandage
+
+> ### {% icon hands_on %} Hands-on: Assembly stats and visualization with Bandage
+>
+> 1. **Bandage Info** {% icon tool %} with the following parameters
+>    - {% icon param-file %} *"Graphical Fragment Assembly"*: the assembly graph dataset produced by
+>      **Unicycler**
+>    - *"Output the information in a single tab-delimited line starting with the graph file"*:
+>      `No`
+>
+> 2. **Bandage Image** {% icon tool %} with the following parameters
+>    - {% icon param-file %} *"Graphical Fragment Assembly"*: the assembly graph dataset produced by
+>      **Unicycler**
+{: .hands_on}
+
+### Check origin of assembled sequences with BLAST
+
+**TO DO** (point out 100% identity to published SARS-CoV-2 sequence of largest
+assembled sequence. Discuss second assembled sequence (*Prevonella* species as
+opportunistic pathogens)
```

Comment:
The *Prevonella* assembly is brought in exclusively by `SRR10971381`.
Could add a question how to discover that, and also link to *Ecology* tutorials for approaches to detect more other species in the data.